services/alerts/src/useAlert.ts

```diff
@@ -0,0 +1,29 @@
+import { useCallback, useContext } from 'react'
+import { AlertsManagerContext } from './AlertsManagerContext'
+import { Alert, AlertOptions } from './types'
+
+export const useAlert = (
+    message: string | ((props: any) => string),
+    options: AlertOptions | ((props: any) => AlertOptions) = {}
+) => {
+    const alertsManager = useContext(AlertsManagerContext)
+
+    const show = useCallback(
```

Comment:
I don't think we should use `useCallback` here and leave it to the app if they want to keep the reference to the function by using `useCallback` itself

Reply:
I'm not 100% sure, but I do think we might have to keep this one to prevent unneeded rerenders in the components that use this hook...